react/features/desktop-picker/components/DesktopPicker.js

```diff
@@ -112,20 +106,19 @@ class DesktopPicker extends Component {
         this._onPreviewClick = this._onPreviewClick.bind(this);
         this._onSubmit = this._onSubmit.bind(this);
         this._updateSources = this._updateSources.bind(this);
+
+        const { desktopSharingSources } = this.props.options;
```

Comment:
I wonder, while you're refactoring, if it's worthwhile to change the prop to desktopSharingSources instead of having a generic options prop.